src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/MissingDependencyNotification.java

```diff
@@ -6,18 +6,19 @@
 import com.intellij.notification.Notification;
 import com.intellij.notification.NotificationType;
 import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+import java.util.Set;
 import org.jetbrains.annotations.NotNull;
 
 public class MissingDependencyNotification extends Notification {
   /**
    * Construct a missing module notification that explains that a module with the given name
```

Comment:
Now that we have a set of module names, this should read:
"...notification which explains that modules with the given names couldn't be found."